dev/com.ibm.ws.microprofile.faulttolerance_repeat_tests/src/com/ibm/ws/microprofile/faulttolerance/fat/repeat/RepeatFaultTolerance.java

```diff
@@ -146,21 +79,19 @@ public static RepeatTests repeatDefault(String server) {
      * @return the RepeatTests rule
      */
     public static RepeatTests repeatAll(String server) {
-        return RepeatTests.with(mp13Features(server))
-                        .andWith(mp20Features(server))
-                        .andWith(ft20metrics11Features(server))
-                        .andWith(mp30Features(server))
-                        .andWith(mp32Features(server))
-                        .andWith(mp33Features(server))
-                        .andWith(mp40Features(server));
+        return repeat(server, TestMode.LITE, MicroProfileActions.MP13, MicroProfileActions.MP20, MicroProfileActions.MP22, MicroProfileActions.MP30, MicroProfileActions.MP32,
+                      MicroProfileActions.MP33, MicroProfileActions.MP40);
     }
 
+    /**
+     * Repeat on MicroProfile 2.0 and above. MP40 will be in LITE mode, the others in FULL mode.
+     *
+     * @param server the server name
+     * @return the RepeatTests rule
+     */
     public static RepeatTests repeat20AndAbove(String server) {
-        return RepeatTests.with(ft20metrics11Features(server).fullFATOnly())
-                        .andWith(mp30Features(server).fullFATOnly())
-                        .andWith(mp32Features(server).fullFATOnly())
-                        .andWith(mp33Features(server).fullFATOnly())
-                        .andWith(mp40Features(server));
+        return repeat(server, TestMode.FULL, MicroProfileActions.MP40, MicroProfileActions.MP20, MicroProfileActions.MP22, MicroProfileActions.MP30, MicroProfileActions.MP32,
+                      MicroProfileActions.MP33);
```

Comment:
I think the design of this method was to repeat for Fault Tolerance 2.0 and above, rather than MP 2.0 and above (because the FT core was rewritten for FT 2.0).

It should not include `MP20` - the rest is correct.

Reply:
doh!

Reply:
fixed